packages/forms-web-app/src/constants.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  constants: {
```

Comment:
We don't need the additional `constants` level in the object and removing it would reduce the number of params needed to access a value.